enhancements/cluster-logging/forwarder-to-otlp.md

```diff
@@ -0,0 +1,201 @@
+---
+title: forward-to-otlp
+authors:
+  - "@jcantrill"
+reviewers:
+  - "@alanconway"
+  - "@cahartma"
+  - "@pavolloffay"
+  - "@periklis"
+  - "@xperimental"
+approvers:
+  - "@alanconway"
+api-approvers: 
+  - "@alanconway"
+creation-date: 2024-04-30
+last-updated: 2024-04-30
+tracking-link:
+  - "https://issues.redhat.com/browse/LOG-4225"
+see-also:
+  - "/enhancements//cluster-logging/cluster-logging-v2-apis.md"
+replaces: []
+superseded-by: []
+---
+
+# Log Forwarder to OTLP endpoint
+
+Spec **ClusterLogForwarder.obervability.openshift/io** to forward logs to an **OTLP** endpoint
+
+## Summary
+
+The enhancement defines the modifications to the **ClusterLogForwarder** spec 
+and Red Hat log collector to allow administrators to collect and forward logs 
+to an OTLP receiver as defined by the [OpenTelemetry Observability framwork](https://opentelemetry.io/docs/what-is-opentelemetry/)
```

Comment:
typo framwork

Reply:
I would be more specific here anre put a link to the OTLP docs https://opentelemetry.io/docs/specs/otlp/ or https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto/collector/logs/v1